diff -urN bulmages/bulmacont/plugins/pluginregistroiva/pluginregistroiva.pro bulmages.new/bulmacont/plugins/pluginregistroiva/pluginregistroiva.pro
--- bulmages/bulmacont/plugins/pluginregistroiva/pluginregistroiva.pro	2007-10-06 20:47:12.000000000 +0200
+++ bulmages.new/bulmacont/plugins/pluginregistroiva/pluginregistroiva.pro	2007-12-25 20:15:15.000000000 +0100
@@ -1,9 +1,7 @@
 # Archivo generado por el gestor qmake de kdevelop.
 # -------------------------------------------------
 
-exists(/usr/bin/distcc) {
-    QMAKE_CXX = distcc g++
-}
+QMAKE_CXX = g++
 
 QT +=  xml
 
diff -urN bulmages/bulmacont/src/src.pro bulmages.new/bulmacont/src/src.pro
--- bulmages/bulmacont/src/src.pro	2007-10-06 20:47:12.000000000 +0200
+++ bulmages.new/bulmacont/src/src.pro	2007-12-25 20:15:23.000000000 +0100
@@ -1,9 +1,7 @@
 # Archivo generado por el gestor qmake de kdevelop.
 # -------------------------------------------------
 
-exists(/usr/bin/distcc) {
-    QMAKE_CXX = distcc g++
-}
+QMAKE_CXX = g++
 
 LANGUAGE = C++
 
diff -urN bulmages/bulmafact/src/src.pro bulmages.new/bulmafact/src/src.pro
--- bulmages/bulmafact/src/src.pro	2007-10-06 20:47:12.000000000 +0200
+++ bulmages.new/bulmafact/src/src.pro	2007-12-25 20:15:39.000000000 +0100
@@ -12,14 +12,7 @@
 
 QMAKE_LIBDIR += ../../installbulmages
 
-#Si existe distcc hacemos compilacion distribuida
-exists( /usr/bin/distcc ) {
-   QMAKE_CXX = distcc g++
-}
-# Lo mismo pero para SuSe (compilado distcc desde la fuente)
-exists( /usr/local/bin/distcc ) {
-   QMAKE_CXX = distcc g++
-}
+QMAKE_CXX = g++
 
 QMAKE_CFLAGS_WARN_OFF += strict_aliasing
 QMAKE_CXXFLAGS_WARN_OFF += strict_aliasing
diff -urN bulmages/bulmalib/src/src.pro bulmages.new/bulmalib/src/src.pro
--- bulmages/bulmalib/src/src.pro	2007-10-06 20:47:12.000000000 +0200
+++ bulmages.new/bulmalib/src/src.pro	2007-12-25 20:15:50.000000000 +0100
@@ -1,12 +1,7 @@
 # Archivo generado por el gestor qmake de kdevelop.
 # -------------------------------------------------
 
-#Probamos de usar distcc
-#!isEmpty( DISTCC_HOSTS ) {
-exists(/usr/bin/distcc){
-    QMAKE_CXX = distcc g++
-}
-#}
+QMAKE_CXX = g++
 
 
 FORMS += logpassbase.ui \
