diff -urN q/bulmages/installbulmages/bgtrml2pdf/bg__init__.py bulmages-0.9.1/bulmages/installbulmages/bgtrml2pdf/bg__init__.py
--- q/bulmages/installbulmages/bgtrml2pdf/bg__init__.py	1970-01-01 01:00:00.000000000 +0100
+++ bulmages-0.9.1/bulmages/installbulmages/bgtrml2pdf/bg__init__.py	2007-03-03 17:20:03.000000000 +0100
@@ -0,0 +1,18 @@
+# trml2pdf - An RML to PDF converter
+# Copyright (C) 2003, Fabien Pinckaers, UCL, FSA
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+from bgtrml2pdf import parseString
diff -urN q/bulmages/installbulmages/bgtrml2pdf/__init__.py bulmages-0.9.1/bulmages/installbulmages/bgtrml2pdf/__init__.py
--- q/bulmages/installbulmages/bgtrml2pdf/__init__.py	2007-02-20 00:32:19.000000000 +0100
+++ bulmages-0.9.1/bulmages/installbulmages/bgtrml2pdf/__init__.py	1970-01-01 01:00:00.000000000 +0100
@@ -1,18 +0,0 @@
-# trml2pdf - An RML to PDF converter
-# Copyright (C) 2003, Fabien Pinckaers, UCL, FSA
-#
-# This library is free software; you can redistribute it and/or
-# modify it under the terms of the GNU Lesser General Public
-# License as published by the Free Software Foundation; either
-# version 2.1 of the License, or (at your option) any later version.
-#
-# This library is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-# Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public
-# License along with this library; if not, write to the Free Software
-# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-
-from trml2pdf import parseString
